src/CMakeLists.txt

```diff
@@ -70,10 +70,11 @@ set_target_properties(fmilibwrapper PROPERTIES POSITION_INDEPENDENT_CODE ON)
 target_include_directories(fmilibwrapper
         PUBLIC
         "$<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/include>"
+        "$<BUILD_INTERFACE:${generatedSourcesDir}>"
```

Comment:
Arn't these internal use only?

Reply:
Ok, but that means that the till now self-contained FMI part of the code-base depends on something non-fmi related. I wonder if we should try to keep the encapsulation.

Reply:
I think the clients only need to use `ExportedState` when they run `export_state` and `import_state` on the fmu, if that was your concern. 

In `libcosim` (in a new PR), `ExportedState` is converted into `serialization::node` in `remote_slave` before it is passed to the higher level. So I don't think the current code-base will be affected as long as they do not use this new state saving feature.

FYI see how this new change is used in `libcosim`:

https://github.com/open-simulation-platform/libcosim/blob/ccca8a9e076c1b2bea8b54c19113828927af546e/src/cosim/proxy/remote_slave.cpp#L226-L262

Reply:
What I was trying to say is that inside proxyfmu there is a library called fmilibwrapper that is a standalone FMI import library which only depend on FMI Library. This PR introduces thrift as a dependency on that module, which breaks software encapsulation. I would prefer to keep the encapsulation, which can be achived by defining the appropriate structure in fmilibwrapper and converting to the thift generated structure in the proxy code.

Mode code, but clearer separation between modules.

Reply:
Ideally, both libcosim and proxyfmu should depend on this common FMI importer.

Reply:
Thanks for explanation. Updated PR according to the suggestion.